README.md

```diff
@@ -1,24 +1,55 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Hello-Rails-React
 
-Things you may want to cover:
+ ## General info
+  Implemented a connection between a Ruby on Rails back-end and React front-end in one application.
 
-* Ruby version
+ ## Built with
+Project is created with:
+ * Ruby On Rails
+ * React-Rails gem
+ * Redux
+ * Postgres
 
-* System dependencies
 
-* Configuration
+## Setup
+To run this project, install it locally using:
+- cd Desktop
+- git clone https://github.com/VuDej/hello-rails-react.git
+- cd hello-rails-react
+- bundle install
+- rails server
+- in your browser open url : https://localhost:3000 to view the app.
+- rails db:create db:migrate db:seed
 
-* Database creation
+## Author
 
-* Database initialization
+👤 **Dejan Vujovic**
 
-* How to run the test suite
+- Github : [@VuDej](https://github.com/VuDej)
+- Twitter: [@DejanVuj](https://twitter.com/DejanVuj)
+- LinkedIn : [@Dejan-Vujovic](https://www.linkedin.com/in/dejan-vujovic-5a0672225/)
 
-* Services (job queues, cache servers, search engines, etc.)
 
-* Deployment instructions
 
-* ...
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/VuDej/School-Library-Ruby/issues/1).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+A special thanks to Microverse.
+
+## 📝 License
+
+This project is [MIT](LICENSE) licensed.
```

Comment:
- [ ] Great job adding the license file, however, the link does not work because it points to the wrong path.  Kindly fix it so it points to the correct file 💪 

![image](https://user-images.githubusercontent.com/57421823/170983598-74e11e2c-a001-42c9-bdc7-7bb1abe4f3e7.png)